bobtemplates/odoo/hooks.py

```diff
@@ -187,8 +187,14 @@ def post_render_readme(configurator, addon_name=""):
         if isinstance(variables["addon.oca"], bool)
         else to_boolean(False, False, variables["addon.oca"])
     )
+    generate_readme = (
+        variables["addon.readme"]
```

Comment:
```suggestion
        variables.get("addon.readme", True)
```
Below as well.

Without this, I think the readme command will fail. I'll have to add a test for that sometime.

Reply:
Indeed. Fixed.